src/components/flyout/CloseButton.tsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+
+import Button from '../button';
+import IconClose from '../../icons/general/IconClose';
+import { bdlGray62 } from '../../styles/variables';
+import './CloseButton.scss';
+
+export interface CloseButtonProps {
+    className?: string;
+    onClick?: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
+}
+
+const CloseButton = ({ className, onClick }: CloseButtonProps) => {
+    return (
+        <Button onClick={onClick} className={classNames('bdl-OverlayHeaderClose', className)}>
```

Comment:
Let's make sure to include `type="button"`, since the default is `submit`, which can cause a full page reload.